edi_storage_oca/migrations/16.0.1.0.0/pre-migrate.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import SUPERUSER_ID, api
+from odoo.tools.sql import column_exists, create_column, rename_column
+
+from odoo.addons.http_routing.models.ir_http import slugify
+
+
+def migrate(cr, version):
```

Comment:
You can use @openupgrade.migrate()
in order to have env directly on the migration function

you can check functionlities of openupgrade here:

https://github.com/OCA/OpenUpgrade/blob/16.0/openupgrade_scripts/scripts/account/16.0.1.2/pre-migration.py#L525-L528

Reply:
Thank @etobella ,the PR is updated as you suggested